lib/libzutil/os/linux/zutil_import_os.c

```diff
@@ -403,6 +403,24 @@ struct sockaddr_un zfs_user_socket = {
 	AF_UNIX, "/run/zfs_user_socket"
 };
 
+static int
+read_full(int fd, void *buf, size_t len)
```

Comment:
In other places we called this `read_all`
```suggestion
read_all(int fd, void *buf, size_t len)
```

Reply:
Done